reoptjl/test/test_job_endpoint.py

```diff
@@ -283,4 +283,20 @@ def test_hybridghp(self):
         r = json.loads(resp.content)
 
         # calculated_ghx_residual_value 117065.83
-        self.assertAlmostEqual(r["outputs"]["GHP"]["ghx_residual_value_present_value"], 117065.83, places=0)
\ No newline at end of file
+        self.assertAlmostEqual(r["outputs"]["GHP"]["ghx_residual_value_present_value"], 117065.83, places=0)
+
+    def test_centralghp(self):
+        post_file = os.path.join('reoptjl', 'test', 'posts', 'central_plant_ghp.json')
+        post = json.load(open(post_file, 'r'))
+
+        # Call http.jl /reopt to run the central plant GHP scenario and get results for defaults from julia checking
+        resp = self.api_client.post('/v3/job/', format='json', data=post)
+        self.assertHttpCreated(resp)
+        r = json.loads(resp.content)
+        run_uuid = r.get('run_uuid')
+
+        resp = self.api_client.get(f'/v3/job/{run_uuid}/results')
+        r = json.loads(resp.content)
+
+        # calculated_ghx_residual_value 117065.83
+        self.assertAlmostEqual(r["outputs"]["Financial"]["lifecycle_capital_costs"], 1046066.8, places=0)
```

Comment:
Do we expect this to pass all the time with the variability in GHX sizing? Also, since we just have a temporary fix which will change costs with rebate $/ton incentives, should we force in that there are no rebate incentives in the inputs in this tests post (just to be certain things don't change later)?

Reply:
@Bill-Becker I updated the hybrid and central GHP tests with a bigger tolerance. Tests seem to be passing. Do you see any other issues?